big-int/src/bigint.hpp

```diff
@@ -33,117 +33,92 @@
 
 namespace libbig
 {
+  constexpr bool POSITIVE = true;
+  constexpr bool NEGATIVE = false;
 class largeInt
 {
   private:
     bool sign;
     std::string number;
 
   public:
-    largeInt()
-    {
-    }
-
-    largeInt(int)
-    {
-    }
-    largeInt(int64_t)
-    {
-    }
-    largeInt(long long)
-    {
-    }
-    largeInt(std::string)
-    {
-    }
+    largeInt();
+
+    explicit largeInt(int);
+    explicit largeInt(long long int);
+    largeInt(std::string);
```

Comment:
Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285290)